### PR TITLE
Update README for dnb-service environment variables

### PR DESCRIPTION
**Pull request by DeanElliott96**
### Description of change

Updated the README for the dnb-service environment variables as we were unsure where these were coming from.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -405,7 +405,7 @@ Data Hub API can run on any Heroku-style platform. Configuration is performed vi
...
 | `DNB_SERVICE_BASE_URL`                                                | No                                          | The base URL of the DNB service.                                                                                                                                                 |
-| `DNB_SERVICE_TOKEN`                                                   | No                                          | The shared access token for calling the DNB service.                                                                                                                             |
+| `DNB_SERVICE_TOKEN`                                                   | No                                          | Access token provided by dnb-service through its djang-rest-framework token authentication.                                                                                                                             |
```
Comment (oliverjwroberts) [inline README.md:408]: "djang" is missing the "o"

**Review by oliverjwroberts — APPROVED** [review]
PR supplements existing documentation for `DNB_SERVICE_BASE_URL` and `DNB_SERVICE_TOKEN` env variables.

Much more informative!

File: README.md (hunk 1 above)
Comment (oliverjwroberts) [reply README.md]: Added a commit to fix the typo so the tests have re-run and completed once you see this.

File: README.md [hunk 2]
```diff
@@ -99,7 +99,7 @@ There is now a `make` command to bring up the three environments on a single doc
    and `DJANGO_SUPERUSER_SSO_EMAIL_USER_ID` the same as `DJANGO_SUPERUSER_EMAIL` in
    data-hub-api .env file otherwise the user may not exist
 
-   You will also need to add `DNB_SERVICE_BASE_URL` and `DNB_SERVICE_TOKEN` to dnb-service/.env in order to use DnB functionality. These credentials can be found in Vault
+   The `DNB_SERVICE_BASE_URL` should match the dnb-service domain you are trying to access (localhost, staging, uat etc). The `DNB_SERVICE_TOKEN` is a token generated by dnb-service using the django-rest-framework token authentication. For localhost, you will need to generate a token on dnb-service for your user. For staging, uat or prod, the current tokens can be found in vault which are associated with an already existing api user. See the [django rest docs](https://www.django-rest-framework.org/api-guide/authentication/#tokenauthentication) for more information on token authentication.
```
Comment (marijnkampf) [inline README.md:102]: Should the info from "For localhost, ... authentication." be included in the DnB service documentation instead?

**Review by marijnkampf — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (DeanElliott96) [reply README.md:102]: Yeah I can update the DNB docs and refer to the docs here.

Comment (DeanElliott96) [reply README.md:102]: PR here in dnb-service: https://github.com/uktrade/dnb-service/pull/168/files. 

I'll update this PR with a link to the dnb-service README once its merged.

Comment (DeanElliott96) [reply README.md]: Added link to dnb-service README.

**Pull request merged**